### ISSUE TITLE
Do you need to downsample the video for correlation image(to calculate the summary image), if you downsampled using ssub,tsub ??

### ISSUE THREAD

**Issue by bashsyr** [flatironinstitute/CaImAn#1158]
Your setup:
 1. Operating System (Linux, MacOS, Windows): Windows
 2. Hardware type (x86, ARM..) and RAM: x86 RAM:64GB
 3. Python Version (e.g. 3.9): 3.10.8
 4. Caiman version (e.g. 1.9.12):1.9.15
 5. Which demo exhibits the problem (if applicable): pipeline_cnmfe
 6. How you installed Caiman (pure conda, conda + compile, colab, ..): usual mamba setup in Windows liek the README
 7. Details: 
 
Hello,
I basically wrote the question in the title.
In #815 it was mentioned that when you downsample min_pnr is automatically adjusted [here](https://github.com/flatironinstitute/CaImAn/blob/87c46b76c06f32bc3a992ebee8c3ec5b90214094/caiman/source_extraction/cnmf/initialization.py#L1225) min_corr is not. But that was in 2020 and I couldn"t find any up to date answers
So my question now is: do I need to downsample the video manually for calculating the cn_filter,pnr in [here](https://github.com/flatironinstitute/CaImAn/blob/87c46b76c06f32bc3a992ebee8c3ec5b90214094/demos/general/demo_pipeline_cnmfE.py#L186) or is it done automatically ?
Thank you so much for your answer

**Comment by EricThomson** [comment]
The code hasn't been changed for these calculations since that issue was opened, so whatever @j-friedrich said then still holds. I will need to go through and look at this.

At some point it would be good to write up a table or list enumerating exactly what you have to keep track of when you subsample (e.g., gSig, gSiz, etc). That code you linked to is pretty hard to parse (for this human), and we should probably try to make it a bit more explicit and readable as a code sprint/refactor goal.
